scripts/build/extend_version_file.py

```diff
@@ -116,18 +113,17 @@ def extend_version_file(repository_root, build_dir):
         formatter_class=argparse.RawDescriptionHelpFormatter,
         description=description)
 
-    parser.add_argument('-b', '--build-dir',
-                        required=True,
-                        action='store',
-                        dest='build_dir',
-                        help="Build directory of the source repository.")
-
     parser.add_argument('-r', '--repository',
                         required=True,
                         action='store',
                         dest='repository',
                         help="Root path of the source repository.")
 
+    parser.add_argument('versionfile',
```

Comment:
shouldn't this argument be ?
```suggestion
    parser.add_argument('--versionfile',
```

Reply:
This is a positional argument.